docker-bake.hcl

```diff
@@ -175,6 +175,7 @@ target "_distro-static" {
     DISTRO_ID = ""
     DISTRO_SUITE = ""
     DISTRO_IMAGE = DISTRO_IMAGE != null && DISTRO_IMAGE != "" ? DISTRO_IMAGE : "debian:bookworm"
+    TEST_ONLY = "0"
```

Comment:
Silly thought; would it make sense to do the reverse, i.e. currently it's opt-out when adding a distro; wondering if it should be "opt-in" to be deployed.

Or we could even have a `CHANNELS` param to allow having upcoming distros in our `test` channel, but not in `stable`.

Reply:
> Or we could even have a `CHANNELS` param to allow having upcoming distros in our test channel, but not in stable.

Yeah this is basically what it does, except less mouthy. `TEST_ONLY` will just deploy to test.

Reply:
Oh! I thought it was "test in CI only, but don't deploy"

FWIW, we had some examples of that in the other repositories, where we test "Fedora Rawhide" and "Debian sid" (testing) in CI.

Those should never, ever be deployed (not even in the "test" channel), but are good to have to validate if upcoming distro releases need changes in packaging.

So that was my thinking behind a "channels" arg; which could be empty if a distro should not be deployed at all